kalite/updates/static/js/updates/bundle_modules/update_videos.js

```diff
@@ -299,7 +299,8 @@ $(function() {
 
 
     if ($("#download_language_selector option").length > 1) {
-        $("#language_choice_titlebar a").attr("onclick", "show_language_selector();");
+        // $('#language_choice_titlebar a').click(show_language_selector);
```

Comment:
Definitely agree that it is better just to show the drop down rather than have the onclick behaviour. Maybe just delete this line instead of commenting it out?